wasmcc/wasmcc_test.go

```diff
@@ -150,6 +152,12 @@ var _ = Describe("Tests for wasmcc simple asset transfer", func() {
 
 func ReadAssetTransferWASMZip() []byte {
 
+	//delete existing zip file
```

Comment:
I know these are tests, but perhaps they should be split out into their own change

Reply:
agree, should be separate commit at least.